README.md

```diff
@@ -44,13 +44,15 @@ more files to this list as you like:
 ```rust,no_run
 extern crate skeptic;
 
+use std::path::PathBuf;
+
 use skeptic::*;
 
 fn main() {
     // Add all markdown files in directory "book/".
     let mut mdbook_files = markdown_files_of_directory("book/");
     // Also add "README.md" to the list of files.
-    mdbook_files.push("README.md".to_owned());
+    mdbook_files.push(PathBuf::from("README.md"));
```

Comment:
you can use `"README.md".into()` practically everywhere instead of `PathBuf::from` making the breaking change less painful and examples a little less verbose.